scene/gui/graph_edit.cpp

```diff
@@ -633,6 +633,7 @@ void GraphEdit::add_child_notify(Node *p_child) {
 		GraphNode *graph_node = Object::cast_to<GraphNode>(graph_element);
 		if (graph_node) {
 			graph_node->connect("slot_updated", callable_mp(this, &GraphEdit::_graph_node_slot_updated).bind(graph_element));
+			graph_node->connect(SNAME("slot_sizes_changed"), callable_mp(this, &GraphEdit::_graph_node_rect_changed).bind(graph_node));
```

Comment:
Why this one uses SNAME and one above doesn't?

Reply:
Probably by accident, removed it.